### DISCUSSION TITLE
Is there a way to transfer local history from Windows 7 machine to Ubuntu 20.04 ?

### DISCUSSION THREAD

**Discussion by deepvision7** [apache/netbeans#5200] [Q&A]
I tried it by exporting the file history using `Tools -> Options -> Export -> Team -> History` from the Windows 7 machine.
But after importing it on Ubuntu 20.04 running in VirtualBox 7 no history appeared.
Probably because of a little bit different path for project files between Windows and Linux. Like on Windows it starts with `d:\home` and on Linux with `/home`. But the rest of the path is the same.
Is there any way to transfer the file history in my situation?

**Comment by Chris2011** [comment]
You can try to copy it by hand from user dir where NetBeans saves all plugins, options, etc. and paste in the other user dir of Ubuntu.

**Reply by deepvision7** [reply]
Yes, this is exactly what I did first, but it didn't work either. The files are in proper places but history won't appear in Netbeans. I suspect it might be due to the little path differences between Windows and Linux (like "d:" and "\\" vs "/" ) which results in different hashes although the files are actually the same.
